src/main/java/io/cdap/plugin/salesforce/plugin/source/streaming/SalesforcePushTopicListener.java

```diff
@@ -51,7 +51,7 @@
    * Timeout of 110 seconds is enforced by Salesforce Streaming API and is not configurable.
```

Comment:
Comment indicates that timeout is not configurable.. how does bumping it up to 1100 seconds help here?

Reply:
@rmstar HttpClient is expecting time is in milli seconds, but the connection timeout which is mentioned is very less and unable to connect. Due to this it is causing timeout exception.

Reply:
So there are 2 timeouts - connection timeout and handshake timeout. Can you update the comment to indicate why you're setting the connection timeout to 1100 seconds, but handshake timeout is 110 seconds (i.e why 1 timeout is 10x the other timeout)?

Reply:
@rmstar HANDSHAKE_TIMEOUT_MS and HANDSHAKE_CHECK_INTERVAL_MS is to wait for sometime to wait after connecting giving a request to the salesforce basically it is waiting whether the handshake happened. But the CONNECTION_TIMEOUT_SECONDS is to give a request call to the salesforce using the HttpClient, as it is having less time, it is unable to complete the request by that time.

Reply:
separate PR has been raised for this by Sanchit. Please discard this PR